src/templates/disease-cell-line.tsx

```diff
@@ -1,60 +1,221 @@
-import React from "react";
-import PropTypes from "prop-types";
-import { graphql } from "gatsby";
+import React, { useState } from "react";
+import { graphql, Link } from "gatsby";
+import { Button, Card, Divider, Flex, Tooltip } from "antd";
+
 import Layout from "../components/Layout";
+import {
+    Clone,
+    DiseaseCellLineFrontmatter,
+    GeneFrontMatter,
+} from "../component-queries/types";
+import { formatCellLineId, getCloneSummary } from "../utils";
+import CloneTable from "../components/CloneTable";
+import Icon from "@ant-design/icons";
+import InfoPanel from "../components/shared/InfoPanel";
+import { primaryColor } from "../style/theme";
+import {
+    DarkThemeGhostButton,
+    DefaultButton,
+} from "../components/shared/Buttons";
+
+const {
+    container,
+    title,
+    extraLargeButton,
+} = require("../style/main-card.module.css");
+const Share = require("../img/share-icon.svg");
+const Arrow = require("../img/arrow.svg");
 
+interface DiseaseCellLineTemplateProps {
+    href: string;
+    cellLineId: string;
+    geneName: string;
+    geneSymbol: string;
+    status: string;
+    snp: string;
+    orderLink: string;
+    certificateOfAnalysis: string;
+    parentalLine: any;
```

Comment:
Why is `parentalLine` `any` here?

Reply:
it was a placeholder that I forgot to go back to, thanks for the catch !